src/server/Document/component.jsx

```diff
@@ -27,14 +27,15 @@ const Document = ({
   const headScript = helmet.script.toComponent();
   const serialisedData = JSON.stringify(data);
   const scriptsAllowed = !isAmp;
+  const noJsTag = !isAmp && { className: 'no-js' };
```

Comment:
I'd call this `noJsHtmlAttrs`, tag seems a misnomer.

Would it also be worth a comment to explain why the noJS class should not be output for AMP?